go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/snapcore/go-gettext
+
+go 1.13
```

Comment:
Silly question - what does "go 1.13" mean here? I looked at the docs and I could not quite find information what consequences this has :/

Reply:
Sure, the doc is quite terse about it.

It currently doesn't force anything. It means that this has been tested in module mode successfully starting go 1.13 (a little bit like a minimum version selection,  but for go stdlib itself).

Upstream sketched up plans so that in the future, it will force (in module mode again) to have this minimum golang version to compile. My guess is that it will be helpful for them in particularly in the Go 2 transitions, if there are some breaking changes, to be able to compile some modules in Go 1 compatiblity mode and others in Go 2, as they told they want to be able to mix Go 1 and Go 2 code. But that's really just a guess :)

Setting it here to 1.13 is fine (even if this module is compatible with older Go stdlib version), because this is the first version with module out of experimental.
Basically, if you still compile with `GOPATH` (which I think snapd is doing on older release and still is today, even in 1.13), this has no impact.

Reply:
Thanks for explaining this!